packages/react-scripts/template/src/app/app.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './app';
```

Comment:
@aneagoie I know lots of people in the community use capitalized file names. However at Rangle our standing practice has been not to do this.  We capitalize the actual component name in the code, but not the file name.

Why?

Because mixed-case file names cause all kinds of problems in teams where people are using a variety of OSs. Recall that:

* filenames in linux are case sensitive
* filenames in OSX preserve the case, but the os allows you to reference them with either case
* filenames in Windows just lose the casing information.

When you end up troubleshooting some weird CI issue only to realize it's because CI's linux container can't find a bunch of files because of casing inconsistencies I thing you'll come around :)